addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1643,11 +1667,11 @@ const Wysiwyg = Widget.extend({
                         // Deselect tables so the applied color can be seen
                         // without using `!important` (otherwise the selection
                         // hides it).
-                        if (this.odooEditor.deselectTable() && hasValidSelection(this.odooEditor.editable)) {
-                            this.odooEditor.document.getSelection().collapseToStart();
-                        }
+                        this.odooEditor.deselectTable();
                         this._updateEditorUI(this.lastMediaClicked && { target: this.lastMediaClicked });
-                        colorpicker.off('color_leave');
+                        if (ev.name !== 'custom_color_picked') {
+                            colorpicker.off('color_leave');
+                        }
```

Comment:
What if you select the color from `o_color_pick_area` that's a `color_picked` event right.
So if I see it correctly, the bug will still occur if you select a color from the `o_color_pick_area` and then hover over any `o_we_color_btn`.

Reply:
When we select color from `o_color_pick_area` it is a `custom_color_picked` event and bug won't occur :smile:.

Reply:
I've updated changes. Please have a look!